src/Collection/DefaultIterator.php

```diff
@@ -361,6 +361,14 @@ public function setSortFunction(callable $sortFunction)
         return $this;
     }
 
+    /**
+     * @return callable
+     */
+    public function getSortFunction(): callable
+    {
+        return $this->sortFunction;
+    }
+
```

Comment:
Please revert this change because extending the API to test internal properties is not an option. Use reflection, an anonymous class or better find a test scenario that do not need access to internal operations.

Reply:
I thought for a while about the related tests, and it will be painful to find a way to "replace" this test with "something useful". I can certainly change the tests to use Reflection. I thought that I would not get away with just adding a getter :)

I found https://github.com/laminas/laminas-ldap/pull/13/files#diff-9ad7ec52897b7302bf9c4dd9b3c4c04e903b948aab795ef87c25db6e6db46078R74 sortTest.php testSorting()
and https://github.com/laminas/laminas-ldap/pull/13/files#diff-9ad7ec52897b7302bf9c4dd9b3c4c04e903b948aab795ef87c25db6e6db46078R116 sortTest.php testCustomSorting()

It looks to me that those test cases cover using the default, and a different sorting function.

So actually, why do we need the test cases that are trying to check the internal value of the private variable `sortFunction` ?

Reply:
I have made the change to use reflection.